raft/multinode.go

```diff
@@ -51,8 +51,8 @@ type MultiNode interface {
 // StartMultiNode creates a MultiNode and starts its background goroutine.
 // The id identifies this node and will be used as its node ID in all groups.
 // The election and heartbeat timers are in units of ticks.
-func StartMultiNode(id uint64) MultiNode {
-	mn := newMultiNode(id)
+func StartMultiNode(id uint64, logger GroupLogger) MultiNode {
```

Comment:
I was thinking that we'd add a `Logger` argument to `CreateGroup` instead of introducing a new type to pass in to `StartMultiNode`. But either way works.